.gradient/prepare-datasets.sh

```diff
@@ -33,24 +11,17 @@ then
 fi
 
 echo "Starting preparation of datasets"
-# symlink exe_cache files
-exe_cache_source_dir="${PUBLIC_DATASET_DIR}/poplar-executables-pytorch-3-1-1"
-symlink-public-resources "${exe_cache_source_dir}" $POPLAR_EXECUTABLE_CACHE_DIR
-# Symlink squad
-symlink-public-resources "${PUBLIC_DATASET_DIR}/squad" "${HF_DATASETS_CACHE}/squad"
-# Symlink OGB Wiki dataset and checkpoint
-symlink-public-resources "${PUBLIC_DATASET_DIR}/ogbl_wikikg2_custom" "${DATASET_DIR}/ogbl_wikikg2_custom"
-
-# symlink local dataset used by vit-model-training notebook
-symlink-public-resources "${PUBLIC_DATASET_DIR}/chest-xray-nihcc-3" "${DATASET_DIR}/chest-xray-nihcc-3"
+cd "$(dirname "$0")"
+python -u ./symlink_datasets_and_caches.py
+cd -
 
 # pre-install the correct version of optimum for this release
 python -m pip install "optimum-graphcore>=0.5, <0.6"
```

Comment:
Should this by `python3`?
Suggest checking the return-code also, if we can't operate without this...